### PR TITLE
feat(frontend): installing Solana SDK

### PR DESCRIPTION
**Pull request by AntonioVentilii**
# Motivation

We install Solana SDK that will come useful for the integration.

Documentation at: https://solana.com/docs/clients/javascript

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
 		"@dfinity/verifiable-credentials": "^0.0.4",
 		"@junobuild/analytics": "^0.0.28",
 		"@metamask/detect-provider": "^2.0.0",
+		"@solana/web3.js": "^2.0.0",
```
Comment (peterpeterparker) [inline package.json:68]: ⚠️ The solana web3js library has been literally compromised yesterday. From what I remember reading it on twitter late in the evening, I think it was older version, something like version v1.9.xxx but, can you please double check this version is safe?

Comment (AntonioVentilii) [reply package.json:68]: on it!

Comment (peterpeterparker) [reply package.json:68]: Prodsec monitors this but, I thought it was better to start on a good fit. Thanks for double checking.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

Adding such a chunk will have an impact on the JS bundle. We gonna need to have a look at the chunking to see how it is impacted and what's the results in term of performance.

**Pull request merged**